UOP1_Project/Assets/Scenes/CharController.unity

```diff
@@ -188,7 +188,7 @@ Transform:
   m_Children:
   - {fileID: 2020721999}
   m_Father: {fileID: 0}
-  m_RootOrder: 5
+  m_RootOrder: 6
```

Comment:
I adjusted the ordering so that `AudioManager `is grouped with the other managers

![image](https://user-images.githubusercontent.com/5694582/95001711-93573300-059a-11eb-8e7d-916962eed289.png)